common/customizable_localization/00_statecraft_lifestyle_custom_loc.txt

```diff
@@ -1,6 +1,4 @@
-﻿
-
-StatecraftGetVassalType = {
+﻿StatecraftGetVassalType = {
```

Comment:
Could you come up with better names for clan rulers?
It's in `Warcraft-Guardians-of-Azeroth-2\common\flavorization\00_title_holders.txt`
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104962155-46845480-59f1-11eb-8bc0-204a7f43c754.png)
![image](https://user-images.githubusercontent.com/46384992/104962351-ac70dc00-59f1-11eb-9ffb-a9d22725e9d2.png)

</details>